ambry-mysql/src/main/java/com/github/ambry/mysql/MySqlMetrics.java

```diff
@@ -38,6 +42,9 @@
   public final Histogram writeTimeMs;
   public final Histogram readTimeMs;
   public final Histogram copyTimeMs;
+  public final Meter writeRate;
+  public final Meter readRate;
+  public final Meter copyRate;
```

Comment:
in practice, rate metrics behave pretty similarly to count metrics (since count metrics are partitioned into count/sec by the metrics sensors). Is there a specific reason for having both?

Reply:
Didn't think counter does that. If so, i will remove the meter.